.github/workflows/on-push.yml

```diff
@@ -43,6 +43,18 @@ jobs:
           npm ci
           npm run build
 
+      - name: Publish infoview-api (if new version)
+        if: ${{ startsWith(github.ref, 'refs/tags/v') && !endsWith(github.ref, '-pre') && matrix.os == 'ubuntu-latest' }}
+        continue-on-error: true
+        run: |
+          [ "$(npm view @leanprover/infoview-api version)" != "$(sed -n 's/^\s*"version":\s*"\(.*\)",\s*/\1/p' lean4-infoview-api/package.json)" ] && (npm publish --workspace=lean4-infoview-api --access=public) || (echo "infoview-api version unchanged")
```

Comment:
Is this `sed` approach really necessary? Won't publishing the InfoView simply fail if the version is unchanged?

Reply:
It does fail yes, so this is superfluous. It only shortens the CI output on failure.

Reply:
In that case, I would prefer a solution without the `sed` invocation :-)

Reply:
just removed it :)